version.py

```diff
@@ -1,9 +1,9 @@
-short_name = "godot"
-name = "Godot Engine"
+short_name = "redot"
+name = "Redot Engine"
 major = 4
 minor = 4
 patch = 0
 status = "dev"
 module_config = ""
-website = "https://godotengine.org"
+website = "https://www.redotengine.org/"
```

Comment:
can we make this https://redotengine.org/ please

Reply:
✅  Updated based on feedback ( changed the `website` url )